packages/api/src/users/users.repository.ts

```diff
@@ -1,22 +1,35 @@
 import { Injectable } from '@nestjs/common';
 
-import type { User } from '@coderscamp/shared/models/user';
+import type { RegisteredUser, User, UserInformation } from '@coderscamp/shared/models/user';
 
-import { PrismaService } from '../prisma/prisma.service';
+import type { GithubUserData } from '../auth/github/github.model';
+import { UserRepositoryService } from '../contracts/user.repository.service';
 
 @Injectable()
 export class UsersRepository {
```

Comment:
I don't undesrtand why we have this class. Why do not use 'UserRepositoryService' (renamed it to UsersRepository) directly?